CMakeLists.txt

```diff
@@ -1200,14 +1200,18 @@ if(BUILD_TEST)
 
   set(MATMUL_TEST_SRCS)
   list(APPEND MATMUL_TEST_SRCS
-    ${NVFUSER_ROOT}/tests/cpp/test_cutlass_scheduler.cpp
     ${NVFUSER_ROOT}/tests/cpp/test_translate_mma.cpp
     ${NVFUSER_ROOT}/tests/cpp/test_matmul.cpp
     ${NVFUSER_ROOT}/tests/cpp/test_matmul_aten_evaluation.cpp
     # ${NVFUSER_ROOT}/tests/cpp/test_matmul_sass.cpp
     ${NVFUSER_ROOT}/tests/cpp/test_matmul_scheduler.cpp
     ${NVFUSER_ROOT}/tests/cpp/test_mma.cpp
   )
+  if (BUILD_CUTLASS AND CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL 12.8)
+    list(APPEND MATMUL_TEST_SRCS
+      ${NVFUSER_ROOT}/tests/cpp/test_cutlass_scheduler.cpp
```

Comment:
Should we put the guard inside the test file instead? @rdspring1

Reply:
It's the include of nvf_cutlass.h caused build failure.

Reply:
Yes and therefore my question. It sounds like a bug that test_cutlass_scheduler.cpp includes nvf_cutlass.h unconditionally. Are we missing something like https://github.com/NVIDIA/Fuser/blob/6a280f0e299b54cb4a015c6c87b7518182334016/csrc/ir/nodes.cpp#L39-L41?